docs/intro/contributing.txt

```diff
@@ -103,26 +113,84 @@ Download the Django source code repository using the following command:
 
     $ git clone https://github.com/django/django.git
 
+Now that you have a local copy of Django, you can install it just like you would
+install any package using ``pip``. The most convenient way to do so is by using
+a *virtual environment* (or virtualenv) which is a feature built into Python
+that allows you to keep a separate directory of installed packages for each of
+your projects so that these don't interfere with each other.
+
+It's a good idea to keep all your virtualenvs in one place, for example in
+``.virtualenvs/`` in your home directory. Create it if it doesn't exist yet:
+
+.. code-block:: console
+
+    $ mkdir ~/.virtualenvs
+
+Now create a new virtualenv by running:
+
+.. code-block:: console
+
+    $ python3 -m venv ~/.virtualenvs/djangodev
+
+The path is where the new environment will be saved on your computer.
+
+If you are on Windows, the commands might look something like:
+
+.. code-block:: none
+
+    > mkdir C:\Users\Name\virtualenvs
+    > C:\Python34\python -m venv C:\Users\Name\virtualenvs\djangodev
```

Comment:
Let's check if python is added to the path on a default install so we don't need to reference C:\Python34